src/components/graph/util/center-graph-button.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import CenterFocusIcon from '@material-ui/icons/CenterFocusStrong';
+import React, { useCallback } from 'react';
+import { ControlButton, useZoomPanHelper } from 'react-flow-renderer';
+
+const CenterGraphButton = ({ selectedNode }) => {
+    const { setCenter } = useZoomPanHelper();
+
+    const focusNode = useCallback(() => {
+        // if no selected node, center on Root
+        const x = selectedNode ? selectedNode.position.x : 0;
```

Comment:
If we have a way to unselect nodes (then to be in a state with no selected node), are we sure that the root node position is always (0,0) ? At first maybe but if it has at least two children, does it have the 0,0 position or a position between its two children (horizontaly at least). Does it depends of draw style chosen ? 
Anyway, I think it's always a bad idea to have raw values like that and hope it will work in all cases.

Reply:
Currently, the position of the root node is (0,0), calculated via the dagre layout, according to the direction and alignment for rank nodes specified for computation. Changing the computation configuration will affect the position of the root node and thus this part of code might be refactored. Anyway, the behaviour of the "center" button when no node is selected is currently being discussed so this behaviour might change in further versions.

Reply:
Ok